docker/test/fasttest/run.sh

```diff
@@ -171,14 +179,14 @@ function run_cmake
     export CCACHE_COMPILERCHECK=content
     export CCACHE_MAXSIZE=15G
 
-    ccache --show-stats ||:
+    ccache_status
```

Comment:
Could `ccache --zero-stats` (next line) also be moved to function `ccache-status`? (which should by the way now be called `compiler_cache_status`

Reply:
No, `--zero-stats` should be executed only once, it's intended.